### PR TITLE
add line-awesome overwrite for fa-comment

### PR DESCRIPTION
**Pull request by treba123**

### CODE REVIEW

File: liqd_product/assets/scss/_line_awesome_overwrites.scss [hunk 1]
```diff
@@ -55,3 +55,8 @@
     font-family: 'FontAwesome', sans-serif;
     content: '\F266'; // lightbulb
 }
+
+.fa-comment:before {
+    font-family: 'FontAwesome', sans-serif;
+    font-style: normal;
```
Comment (fuzzylogic2000) [inline liqd_product/assets/scss/_line_awesome_overwrites.scss]: Don't you need the content anymore?

Comment (treba123) [reply liqd_product/assets/scss/_line_awesome_overwrites.scss]: No, it already comes from somewhere :/ Not sure, philli probably knows why

Comment (phillimorland) [reply liqd_product/assets/scss/_line_awesome_overwrites.scss]: content wasn't changing icon, was set with classname, but i've just seen now it was meant to be a lightbulb not a comment, which i didnt know... maybe we need to find updated lightbulb content code?

Comment (phillimorland) [reply liqd_product/assets/scss/_line_awesome_overwrites.scss]: ahh thats why it was in the overwrites, god im slow today, sorry! is it meant to be a lightbulb?

Comment (treba123) [reply liqd_product/assets/scss/_line_awesome_overwrites.scss]: Oups, guess I shouldn't have pushed it already :)

Comment (phillimorland) [reply liqd_product/assets/scss/_line_awesome_overwrites.scss]: Yeah I don't get why its a lightbulb? old designs are archived on zeplin so I can't access them, we can ask Sandy on Monday

Comment (fuzzylogic2000) [reply liqd_product/assets/scss/_line_awesome_overwrites.scss]: Haha, it's all fine! I think it's meant to be a comment and if it's back now without the content (which I don't get), we're fine!

**Pull request merged**